web/src/filter.ts

```diff
@@ -1571,7 +1571,11 @@ export class Filter {
 
     is_conversation_view(): boolean {
         const term_type = this.sorted_term_types();
-        if (_.isEqual(term_type, ["channel", "topic"]) || _.isEqual(term_type, ["dm"])) {
+        if (
+            _.isEqual(term_type, ["channel", "topic", "with"]) ||
+            _.isEqual(term_type, ["channel", "topic"]) ||
+            _.isEqual(term_type, ["dm"])
```

Comment:
`["dm", "with"]` is not needed because for `dm`, the `with` operator is removed by `_fix_redundant_with_dm`

Reply:
OK. We There's a possibly reasonable follow-up direction that we should have the `with` operator disappear after being processed. Maybe start a #frontend discussion about the trade-offs there?